stackslib/src/net/rpc.rs

```diff
@@ -553,7 +553,7 @@ impl ConversationHttp {
                         self.handle_request(req, node)
                     })?;
 
-                    info!("Handled StacksHTTPRequest";
+                    debug!("Handled StacksHTTPRequest";
```

Comment:
Can you please revert this?  We use the `info!` level at the Foundation so we can assess the node's HTTP performance behavior.

Reply:
Sure